packages/api-key-service/src/functions/create.api-key.ts

```diff
@@ -0,0 +1,38 @@
+import { httpHandler, ValidationException } from '@homeservenow/serverless-aws-handler'
+import { plainToClass } from 'class-transformer'
+import { validate } from 'class-validator'
+import { ApiKeyDto } from '@/dto'
+import { ApiKeyModel } from '@/models'
+import { createApiKey as create } from '@/services'
+import { authorised, decodeToken } from '@reapit/node-utils'
+
+export const createApiKey = httpHandler<ApiKeyDto, ApiKeyModel>({
+  serialise: {
+    input: (event): ApiKeyDto => {
+      authorised(event)
+      const customer = decodeToken(event.headers['reapit-connect-token'] as string)
```

Comment:
For decoding tokens you can use this https://github.com/reapit/foundations/blob/master/packages/connect-session/src/utils/verify-decode-id-token.ts#L79

Usage `import {connectSessionVerifyDecodeIdToken } from '@reapit/connect-session'`

Reply:
Returns this interface https://github.com/reapit/foundations/blob/master/packages/connect-session/src/types.ts#L22

Reply:
deleted